<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>579</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Tarsnap</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/logos/tarsnap-icon-big.png</normaloff>:/logos/tarsnap-icon-big.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="centralWidgetLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="mainContentSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="TTabWidget" name="mainTabWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>350</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">#mainTabWidget &gt; QTabBar::tab {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
					stop: 0 rgba(85,164,216,225), stop: 0.8 rgba(85,164,216,225),
					stop: 0.9 rgba(46,141,204,255), stop: 1.0 rgba(46,141,204,255));
border: 1px solid #ECF0F1;
border-bottom: none;
border-top-left-radius: 2px;
border-top-right-radius: 2px;
padding: 5px 3px 5px 3px;
width: 55px;
height: 18px;
font-size: 13px;
margin-left: 2px;
color: white;
}

#mainTabWidget &gt; QTabBar::tab:last {
margin-right: 5px;
}

#mainTabWidget &gt; QTabBar::tab:selected {
background: rgba(64,152,212,255);
border: 1px solid white;
border-bottom: none;
}

#mainTabWidget::pane { /* The tab widget frame */
border-top: none;
}

#mainTabWidget::tab-bar {
left: 5px; /* move to the right by 5px */
}

#mainTabWidget::tab {
border-radius: 2px;
}

#mainTabWidget
{
border-radius: 2px;
}
</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="largeLogoUrl">
        <url>
         <string>qrc:/logos/tarsnap-header-h29.png</string>
        </url>
       </property>
       <property name="smallLogoUrl">
        <url>
         <string>qrc:/logos/tarsnap-icon-h29.png</string>
        </url>
       </property>
       <property name="largeLogoFilename" stdset="0">
        <string/>
       </property>
       <property name="smallLogoFilename" stdset="0">
        <string/>
       </property>
       <widget class="BackupTabWidget" name="backupTabWidget">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <attribute name="title">
         <string>Backup</string>
        </attribute>
        <attribute name="toolTip">
         <string>Switch to Backup tab &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
        </attribute>
       </widget>
       <widget class="ArchivesTabWidget" name="archivesTabWidget">
        <attribute name="title">
         <string>Archives</string>
        </attribute>
        <attribute name="toolTip">
         <string>Switch to Archives tab &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
        </attribute>
       </widget>
       <widget class="JobsTabWidget" name="jobsTabWidget">
        <attribute name="title">
         <string>Jobs</string>
        </attribute>
        <attribute name="toolTip">
         <string>Switch to Jobs tab &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
        </attribute>
       </widget>
      </widget>
      <widget class="TTextView" name="journalLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>Journal log</string>
       </property>
       <property name="styleSheet">
        <string notr="true">#journalLog
{
background: transparent;
font-family: Monospace, Monaco;
font-size: 12px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="StatusBarWidget" name="statusBarWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Backup">
    <property name="title">
     <string>&amp;Backup</string>
    </property>
    <addaction name="actionBrowseItems"/>
    <addaction name="actionAddFiles"/>
    <addaction name="actionAddDirectory"/>
    <addaction name="actionClearList"/>
    <addaction name="separator"/>
    <addaction name="actionBackupNow"/>
    <addaction name="actionBackupMorphIntoJob"/>
   </widget>
   <widget class="QMenu" name="menu_Archives">
    <property name="title">
     <string>&amp;Archives</string>
    </property>
    <addaction name="actionInspect"/>
    <addaction name="actionRestore"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionFilterArchives"/>
   </widget>
   <widget class="QMenu" name="menu_Jobs">
    <property name="title">
     <string>&amp;Jobs</string>
    </property>
    <addaction name="actionJobInspect"/>
    <addaction name="actionJobRestore"/>
    <addaction name="actionJobDelete"/>
    <addaction name="separator"/>
    <addaction name="actionJobBackup"/>
    <addaction name="actionBackupAllJobs"/>
    <addaction name="actionAddJob"/>
    <addaction name="actionFilterJobs"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionRefreshAccount"/>
    <addaction name="actionStopTasks"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionMinimize"/>
    <addaction name="actionZoom"/>
    <addaction name="actionFullScreen"/>
    <addaction name="separator"/>
    <addaction name="actionGoBackup"/>
    <addaction name="actionGoArchives"/>
    <addaction name="actionGoJobs"/>
    <addaction name="actionShowJournal"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionTarsnapWebsite"/>
    <addaction name="separator"/>
    <addaction name="actionShowConsoleLog"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Backup"/>
   <addaction name="menu_Archives"/>
   <addaction name="menu_Jobs"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/reload.png</normaloff>:/icons/reload.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Reload archives list from Tarsnap server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionClearList">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/circle-x.png</normaloff>:/icons/circle-x.png</iconset>
   </property>
   <property name="text">
    <string>Clear list</string>
   </property>
   <property name="toolTip">
    <string>Clear list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionBrowseItems">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/magnifying-glass.png</normaloff>:/icons/magnifying-glass.png</iconset>
   </property>
   <property name="text">
    <string>Open File Browser</string>
   </property>
   <property name="toolTip">
    <string>Set files and directories for the Backup tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGoBackup">
   <property name="text">
    <string>Go to Backup</string>
   </property>
   <property name="toolTip">
    <string>Switch to Backup pane</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionGoArchives">
   <property name="text">
    <string>Go to Archives</string>
   </property>
   <property name="toolTip">
    <string>Switch to Archives pane</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionGoJobs">
   <property name="text">
    <string>Go to Jobs</string>
   </property>
   <property name="toolTip">
    <string>Switch to Jobs pane</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionAddJob">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/hard-drive.png</normaloff>:/icons/hard-drive.png</iconset>
   </property>
   <property name="text">
    <string>Add Job</string>
   </property>
   <property name="toolTip">
    <string>Add a new job</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRefreshAccount">
   <property name="text">
    <string>Update account details</string>
   </property>
   <property name="toolTip">
    <string>Update Tarsnap account details from website</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionBackupAllJobs">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/cloud-upload-multi.png</normaloff>:/icons/cloud-upload-multi.png</iconset>
   </property>
   <property name="text">
    <string>Backup all Jobs</string>
   </property>
   <property name="toolTip">
    <string>Backup all jobs now</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionShowJournal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/caret-top.png</normaloff>
     <normalon>:/icons/caret-bottom.png</normalon>:/icons/caret-top.png</iconset>
   </property>
   <property name="text">
    <string>Show Journal</string>
   </property>
   <property name="toolTip">
    <string>Show/hide Journal log &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete archive(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionInspect">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>Inspect</string>
   </property>
   <property name="toolTip">
    <string>Inspect archive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/cloud-download.png</normaloff>:/icons/cloud-download.png</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
   <property name="toolTip">
    <string>Restore Archive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionJobRestore">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/cloud-download.png</normaloff>:/icons/cloud-download.png</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
   <property name="toolTip">
    <string>Restore latest backup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionJobInspect">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>Inspect</string>
   </property>
   <property name="toolTip">
    <string>Show job details</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionJobBackup">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/cloud-upload.png</normaloff>:/icons/cloud-upload.png</iconset>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
   <property name="toolTip">
    <string>Execute backup for job(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionJobDelete">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete job</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionBackupMorphIntoJob">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/hard-drive.png</normaloff>:/icons/hard-drive.png</iconset>
   </property>
   <property name="text">
    <string>Morph into Job</string>
   </property>
   <property name="toolTip">
    <string>Create Job from current Backup list</string>
   </property>
  </action>
  <action name="actionFilterArchives">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/magnifying-glass.png</normaloff>:/icons/magnifying-glass.png</iconset>
   </property>
   <property name="text">
    <string>Filter and search</string>
   </property>
   <property name="toolTip">
    <string>Search and filter Archives list  &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionBackupNow">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/cloud-upload.png</normaloff>:/icons/cloud-upload.png</iconset>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
   <property name="toolTip">
    <string>Backup now &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionStopTasks">
   <property name="text">
    <string>Stop tasks</string>
   </property>
   <property name="toolTip">
    <string>Stop, interrupt or clear queued tasks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionFilterJobs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/magnifying-glass.png</normaloff>:/icons/magnifying-glass.png</iconset>
   </property>
   <property name="text">
    <string>Filter and search</string>
   </property>
   <property name="toolTip">
    <string>Search and filter Jobs list  &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/file.png</normaloff>:/icons/file.png</iconset>
   </property>
   <property name="text">
    <string>Add file(s)</string>
   </property>
   <property name="toolTip">
    <string>Browse for files to add to the Backup list</string>
   </property>
  </action>
  <action name="actionAddDirectory">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Add directory</string>
   </property>
   <property name="toolTip">
    <string>Browse for directory to add to the Backup list</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Tarsnap</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionTarsnapWebsite">
   <property name="text">
    <string>Tarsnap Website</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enter Full Screen</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+F</string>
   </property>
  </action>
  <action name="actionShowConsoleLog">
   <property name="text">
    <string>Show Console Log</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Show Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TTabWidget</class>
   <extends>QTabWidget</extends>
   <header>TTabWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTextView</class>
   <extends>QPlainTextEdit</extends>
   <header>TTextView.h</header>
  </customwidget>
  <customwidget>
   <class>StatusBarWidget</class>
   <extends>QWidget</extends>
   <header>widgets/statusbarwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BackupTabWidget</class>
   <extends>QWidget</extends>
   <header>widgets/backuptabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArchivesTabWidget</class>
   <extends>QWidget</extends>
   <header>widgets/archivestabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JobsTabWidget</class>
   <extends>QWidget</extends>
   <header>widgets/jobstabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
